packages/client/src/components/auth/Login.component.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, FC } from 'react';
+import * as firebaseauth from '@firebase/auth';
+import { TextField, Button, Box, Typography, Dialog, DialogTitle, DialogActions } from '@mui/material';
+
+interface LoginComponentProps {
+  onLoginSuccess: (token: string) => void;
+  auth: firebaseauth.Auth;
+}
+
+// Login Page Component
+const LoginComponent: FC<LoginComponentProps> = ({ auth, onLoginSuccess }) => {
```

Comment:
For consistency with the project, having things exported via `export const LoginComponent` instead of having a default export